ansible_collections/azureredhatopenshift/cluster/tasks/upgrade_cluster.yaml

```diff
@@ -0,0 +1,241 @@
+# This must be invoked in a loop over the upgrade variable:
+# - name: upgrade_cluster | Begin upgrade process
+#   loop: "{{ upgrade }}"
+#   when: upgrade is defined
+#   ansible.builtin.include_tasks:
+#     file: ../../tasks/upgrade_cluster.yaml
```

Comment:
Same as above

Reply:
This is documentation on how to include this particular task file.